indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -92,6 +92,18 @@ def _process(self):
                                 form=form, is_edit=True)
 
 
+class RHSortEmailTemplates(RHManageAbstractsBase):
+    """Sort e-mail templates according to the order provided by the client."""
+
+    def _process(self):
+        new_order = [AbstractEmailTemplate.get(tpl_id) for tpl_id in request.json['sort_order']]
```

Comment:
Well, I'm not expecting us to have 1000 e-mail templates, but I'll see if I find something better that doesn't look awful.